crates/store/src/state.rs

```diff
@@ -52,8 +53,12 @@ struct InnerState {
 
 /// The rollup state
 pub struct State {
+    /// The underlying database.
     db: Arc<Db>,
```

Comment:
I would update the comment to something like: "The database which stores block headers, nullifiers, notes, and the latests states of accounts."